### DISCUSSION TITLE
Wait x days unti install updates and keep x number of images

### DISCUSSION THREAD

**Discussion by MH1984-RPI** [containrrr/watchtower#2011] [Feature Requests]
I would like to request 2 features if possible.

Wait x days until install an update. For instance wait 7 days to install like to think stable at this point if not replaced.

Used Watchtower a few times in the past.

Keep x number of images. For instance 2 current/previous image.

**Comment by huyz** [comment]
Cool, the first request was exactly what I've been wanting as welll: for each container, I'd like to put a label that says how many hours and/or days an update must be the latest before it is installed.
This would ensure that there isn't a quick patch that needs to be released (in essence, let other users be the guinea testers / beta pigs)
